Basic/ClientDriven/Assets/Scripts/ClientPlayerMove.cs

```diff
@@ -45,6 +49,8 @@ public override void OnNetworkSpawn()
         if (!IsOwner)
         {
             enabled = false;
+            m_CharacterController.enabled = false;
```

Comment:
mind adding a quick explanation on the why we did this? Those lines are part of the "magic" that makes Character Controller work, this should be highlighted.